app/(gcforms)/[locale]/(form administration)/form-builder/[id]/responses/[statusFilter]/actions.ts

```diff
@@ -149,12 +167,14 @@ export const getSubmissionsByFormat = async ({
   format = DownloadFormat.HTML,
   lang,
   revalidate = false,
+  extraTranslations = {},
 }: {
   formID: string;
   ids: string[];
   format: DownloadFormat;
   lang: Language;
   revalidate?: boolean;
+  extraTranslations?: { [key: string]: { en: string; fr: string } };
```

Comment:
Is there a reason you couldn't use `serverTranslation` wherever you needed to access these?